pkg/terra/utils.go

```diff
@@ -89,10 +97,40 @@ func ContractConfigToOCRConfig(in []byte) ([]byte, error) {
 	if len(in) != 33 {
 		return nil, fmt.Errorf("invalid config length: expected 33 got %d", len(in))
 	}
-	const byteWidth128 = 16
-	padding := make([]byte, 8) // padding to convert int to 192
 	version := in[0:1]
-	min128 := in[1 : byteWidth128+1]
-	max128 := in[1+byteWidth128:]
-	return bytes.Join([][]byte{version, padding, min128, padding, max128}, []byte{}), nil
+	min, err := parse128BitSignedInt(in[1 : byteWidth128+1])
+	if err != nil {
+		return nil, err
+	}
+	max, err := parse128BitSignedInt(in[1+byteWidth128:])
+	if err != nil {
+		return nil, err
+	}
+	minBytes, err := median.ToBytes(min)
```

Comment:
Hmm, is there an opportunity for us to use [OnchainConfig.Encode()](https://github.com/smartcontractkit/libocr/blob/5b957c8342763a061fcbd6806ef98351f3cc7f93/offchainreporting2/reportingplugin/median/median.go#L54) directly?